SU-SWS/stanford_profile_helper
# D8CORE-2028 Change "People" to "Users" in the admin toolbar

# READY FOR REVIEW

# Summary
- Alter routes and page names from "People" to "Users"

# Need Review By (Date)
- 5/19

# Urgency
- low

# Steps to Test
1. checkout this branch and the branch in https://github.com/SU-SWS/stanford_profile/pull/414
2. `drush cim -y`
3. view the admin toolbar and verify the "People" menu item is now "Users"

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
